compatibility/features/cdata/cdata.ts

```diff
@@ -0,0 +1,9 @@
+import assert from 'assert'
+import { Given } from '../../../src'
+
+Given(
+  'I have {int} <![CDATA[cukes]]> in my belly',
```

Comment:
I can't find the matching expectation for this one.

Reply:
This runs against the corresponding CCK feature https://github.com/cucumber/compatibility-kit/tree/main/devkit/samples/cdata

Reply:
But the CCK doesn't contain XML to assert against. I would expect this outcome:

https://github.com/cucumber/cucumber-junit-xml-formatter/blob/main/testdata/cdata.feature.xml

Reply:
Ah okay, so that's a test case we can add to the JUnit formatter for now, and then once it moves to the polyglot repo it will have that test anyway.

Reply:
👍

Reply:
Pushed.